packages/interface/src/utils/state.ts

```diff
@@ -21,14 +21,15 @@ export const useRoundState = (roundId: string): ERoundState => {
     return ERoundState.VOTING;
   }
 
+  // TODO commented out for testing results
+  // if (isAfter(now, round.votingEndsAt)) {
+  //   return ERoundState.TALLYING;
+  // }
+
+  // TODO: how to collect tally.json url
```

Comment:
are we going to implement this part in this PR?
I think the status could be
```
if after(votingEndsAt) && fetch(tallyFile):
    return ERoundState.RESULTS
if after(votingEndsAt) && !fetch(tallyFile):
    return ERoundState.TALLYING
```
the tallyFile url is included in the round metadata file.

Reply:
let's fix in a separate PR?

Reply:
I tried the approach with fetching but it will leave so many errors in the console when there's no tally file. best to wait for the tally handlers imo